src/apis/postRequest.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 import axiosInstance from './axiosInstance';
 
+//Tanstack-query로 리팩토링 하기 전 사용하던 POST 요청 추상 메서드
 interface PostRequestResult<T> {
```

Comment:
[참고] 더 이상 사용되지 않아 폐기 예정인 코드들은 [jsdoc](https://jsdoc.app/tags-deprecated) `@deprecated` 어노테이션을 붙이면 IDE에서 표시해줘서 편리합니다!

```suggestion
/**
 * Tanstack-query로 리팩토링 하기 전 사용하던 POST 요청 추상 메서드
 * @deprecated
 */
interface PostRequestResult<T> {
```